CMakeLists.txt

```diff
@@ -197,7 +204,11 @@ target_sources(mold PRIVATE
   macho/tapi.cc
   macho/yaml.cc
   third-party/rust-demangle/rust-demangle.c
-  ${CMAKE_CURRENT_BINARY_DIR}/git-hash.cc)
+  git-hash.cc)
+
+if(MSVC)
+    target_compile_options(mold PRIVATE /bigobj)
```

Comment:
Is this option needed?

Reply:
Yes, else I wouldn't have added it 😉
On a more serious note, MSVC only supports up to 64k addressable sections by default, `/bigobj` raises this limit. It's used relatively frequently.